lib/systems/default.nix

```diff
@@ -36,6 +36,7 @@ rec {
       config = parse.tripleFromSystem final.parsed;
       # Determine whether we can execute binaries built for the provided platform.
       canExecute = platform:
+        final.isAndroid == platform.isAndroid &&
```

Comment:
This doesn't make sense to me.

Reply:
Do you have better idea? It would be better for this to check libc too btw.

Reply:
Musl and glibc can both be executed on the same machine.

What if this is about linux.musl vs linux.glibc? Then this would evaluate to false, right? It should evaluate to true in that case, that's my main point.

Reply:
This is about android bionic libc binary trying to be executed on GNU/Linux host, you can't do that without having android linker and bionic libc on host.

Reply:
I understand. But isCompatible should return true when it's 64-bit to 32-bit 'normal' linux

Reply:
In this case it will

Reply:
Sorry, I guess I am still sleepy :) you're right